Documentation/Security/GuidelinesEditors/BackendAccess.rst.txt

```diff
@@ -85,7 +85,4 @@ someone coming from an unauthorized IP address.
 
 Implementing this additional login limitation is the responsibility of
 the person or company who hosts and/or administers the TYPO3 site.
-Discuss the options with it and keep in mind that this is usually
-a 10 minutes job for someone who knows TYPO3 (the configuration
-directive :ref:`enabledBeUserIPLock <security-global-typo3-options-enabledBeUserIPLock>`
-does exactly this.
+Discuss the options with them.
```

Comment:
![image](https://user-images.githubusercontent.com/999146/102572103-cc289200-411d-11eb-97f2-c0b5b280c6b6.png)

Reply:
I don't know if "it" is used for a "person", you should be sure about it if you use it.

Reply:
> The options should be discussed with the administrating person or company ...